index.ts

```diff
@@ -642,7 +645,7 @@ collect.set('isNewRepoTemplate', [
 const getUsername = (): string | undefined => document.querySelector('meta[name="user-login"]')?.getAttribute('content')!;
 
 /** Drop leading and trailing slashes */
```

Comment:
update the comment?

Reply:
Hmm I think it's fine? Not _super_ clear though